nix/modules/home-manager/vira.nix

```diff
@@ -68,9 +68,16 @@ in
         ProcessType = "Background";
         StandardOutPath = "${config.home.homeDirectory}/Library/Logs/vira.log";
         StandardErrorPath = "${config.home.homeDirectory}/Library/Logs/vira.log";
-        EnvironmentVariables = {
-          PATH = "${makeBinPath cfg.extraPackages}:${config.home.sessionVariables.PATH or ""}";
```

Comment:
Removed `config.home.sessionVariables.PATH or “”`; I don’t think its a common pattern, at least it is [not used](https://github.com/search?q=repo%3Anix-community%2Fhome-manager+sessionVariables.PATH&type=code) in any of the services defined in home-manager